web-client/src/presenter/computeds/formattedCaseDetail.js

```diff
@@ -256,7 +256,15 @@ export const formattedCaseDetail = (get, applicationContext) => {
   );
 
   result.formattedPendingDocketEntriesOnDocketRecord = result.formattedDocketEntriesOnDocketRecord.filter(
-    d => d.pending && applicationContext.getUtilities().isServed(d),
+    docketEntry =>
+      (docketEntry.pending &&
+        applicationContext.getUtilities().isServed(docketEntry)) ||
+      (docketEntry.pending &&
+        applicationContext
+          .getConstants()
+          .UNSERVABLE_EVENT_CODES.find(
+            unservableCode => unservableCode === docketEntry.eventCode,
+          )),
```

Comment:
Just curious, could we not make use of the already existing logic in `shared/src/business/entities/DocketEntry.js`

Reply:
good catch, we will fix that